### PR TITLE
Initial migration to v1

### PR DESCRIPTION
**Pull request by brangi**
## Description
issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/7658

## Testing done
Locally

## Screenshots
Screen Shot 2020-06-08 at 1 00 51 PM

Screen Shot 2020-06-08 at 1 00 24 PM
Screen Shot 2020-06-08 at 1 01 21 PM

## Acceptance criteria
- [x] Migrate Facility locator front end to v1 Facilities VA API

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by ethanteague — APPROVED** [review]
LGTM

**Review by jhonny218 — APPROVED** [review]
LGTM

File: src/applications/facility-locator/tests/02-breadcrumbs.e2e.spec.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ const Timeouts = require('../../../platform/testing/e2e/timeouts.js');
 const FacilityHelpers = require('./facility-helpers');
 
 module.exports = E2eHelpers.createE2eTest(client => {
-  client.url(`${E2eHelpers.baseUrl}/find-locations/`);
+  // Don't know why using the testing instance does not load
+  // Use staging ?
+  client.openUrl(`https://staging.va.gov/find-locations/`);
```
[the remark this answers is not in the document]
Comment (brangi) [reply src/applications/facility-locator/tests/02-breadcrumbs.e2e.spec.js:8]: I know the bigger issue, but tests are very simple and already legacy code. However, I am going to dig in a little more and check why the testing instance does not get results using v1 on testing instance when this is already in staging and production

File: src/applications/facility-locator/tests/api-url-parameters.unit.spec.js [hunk 2]
```diff
@@ -52,7 +52,7 @@ describe('Locator url and parameters builder', () => {
     );
     let test = `${result.url}?${result.params}`;
     expect(test).to.eql(
-      'https://dev-api.va.gov/v0/facilities/va?bbox[]=-118.9939&bbox[]=33.3044&bbox[]=-117.4939&bbox[]=34.8044&type=health&page=1&per_page=20',
+      'https://dev-api.va.gov/v1/facilities/va?bbox[]=-118.9939&bbox[]=33.3044&bbox[]=-117.4939&bbox[]=34.8044&type=health&page=1&per_page=20',
```
[the remark this answers is not in the document]
Comment (brangi) [reply src/applications/facility-locator/tests/api-url-parameters.unit.spec.js]: yeah the locator also mocks API responses - https://github.com/department-of-veterans-affairs/vets-website/blob/07613f4eec00553575b8369c06d189e0fa1af2cc/src/applications/facility-locator/tests/facility-helpers.js#L220 - however this test is only building the API url (path, parameters)  to verify a change combining ccp and va option requests

**Pull request merged**